pytket/docs/changelog.rst

```diff
@@ -7,6 +7,7 @@ Unreleased
 * Support classical transforms and predicates, and QASM registers, with up to 64
   bits. Add an attribute to the pytket module to assert this.
 * Add `PauliSynthStrat.Greedy` strategy to `TermSequenceBox`.
+* Update version requirements on dependencies, including allowing numpy 2.0.
```

Comment:
Maybe we should specify this to make sure it is understood what this means?

Reply:
Do you mean give the exact new version requirements on all the packages? Happy to do that.

Reply:
I mean to point out that there are no upper bounds any more for all packages. (In case any new updates break something it should be clear that it might be possible to fix it by downgrading dependencies)